examples/grpc_example/MODULE.bazel

```diff
@@ -11,57 +11,43 @@ local_path_override(
     path = "../..",
 )
 
-bazel_dep(name = "cgrindel_bazel_starlib", version = "0.25.2")
-bazel_dep(name = "bazel_skylib", version = "1.7.1")
+# Required by rules_swift_package_manager but not pulled in for some reason:
+bazel_dep(name = "cgrindel_bazel_starlib", version = "0.25.1")
```

Comment:
We need to understand this before we merge.

Reply:
I know what it is. I create [an issue](https://github.com/cgrindel/rules_swift_package_manager/issues/1548) to remove the dependency.